keyboards/pica40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layer_number {
+  _QWERTY = 0,
```

Comment:
No need to specify 0 here. 
```suggestion
  _QWERTY,
```